tools/test/test_actions_local_runner.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+import sys
+import contextlib
+import io
+from typing import List, Dict, Any
+
+from tools import actions_local_runner
+
+if sys.version_info < (3, 8):
+    # actions_local_runner uses asyncio features not available in 3.6, and
+    # IsolatedAsyncioTestCase was added in 3.8, so skip testing on
+    # unsupported systems
+    sys.exit(0)
```

Comment:
Is there a better way to skip the test besides using `sys.exit`? Wouldn't this cause any attempts to `import` this file to fail on older Python versions?

Reply:
Yeah you're right, I thought defining it dynamically would mess with `unittest`'s discovery but it works fine apparently